projects/gigya/src/auth/facade/gigya-auth.service.ts

```diff
@@ -0,0 +1,71 @@
+import { Injectable } from '@angular/core';
+import { Store } from '@ngrx/store';
+import {
+  AuthService,
+  StateWithAuth,
+  AuthActions,
+  OCC_USER_ID_CURRENT,
+} from '@spartacus/core';
+import { GigyaAuthActions } from '../store/actions';
+import { take } from 'rxjs/operators';
+
+declare var gigya: any;
```

Comment:
Please avoid referencing gigya directly. Best to use WindowRef service and it's `nativeWindow`. Just to avoid problems with SSR and also to avoid declaring global `gigya` variables.

Reply:
Done